webgpu.h

```diff
@@ -427,6 +437,13 @@ typedef enum WGPUColorWriteMask {
 } WGPUColorWriteMask;
 typedef WGPUFlags WGPUColorWriteMaskFlags;
 
+typedef enum WGPUMappingFlag {
```

Comment:
This may not be the right place to bikeshed, but I didn't see the PR to add this to the spec yet.

I think we should not have the "Flag" suffix here as we don't have it on WGPUShaderStage, WGPUTextureUsage, WGPUBufferUsage, etc. It also makes the `WGPUMappingFlagFlags` typedef kinda ugly.

Maybe WGPUMapMode and WGPUMapModeFlags ?